src/test/java/edu/hm/hafner/analysis/parser/ClangAnalyzerPlistParserTest.java

```diff
@@ -0,0 +1,66 @@
+package edu.hm.hafner.analysis.parser;
+
+import org.junit.jupiter.api.Test;
+
+import edu.hm.hafner.analysis.AbstractParserTest;
+import edu.hm.hafner.analysis.Report;
+import edu.hm.hafner.analysis.Severity;
+import static edu.hm.hafner.analysis.assertj.Assertions.*;
+
+import edu.hm.hafner.analysis.assertj.SoftAssertions;
+
+
+/**
+ * Tests for {@link TaglistParser}.
+ */
+class ClangAnalyzerPlistParserTest extends AbstractParserTest {
+    ClangAnalyzerPlistParserTest() {
+        super("clang-analyzer-test.txt");
+    }
+
+    @Override
+    protected ClangAnalyzerPlistParser createParser() {
+        return new ClangAnalyzerPlistParser();
+    }
+
+    @Override
+    protected void assertThatIssuesArePresent(final Report report, final SoftAssertions softly) {
+        softly.assertThat(report).hasSize(3);
+
+        softly.assertThat(report.get(0))
+                .hasFileName("/src/main.c")
+                .hasSeverity(Severity.WARNING_NORMAL)
+                .hasMessage("1st function call argument is an uninitialized value")
+                .hasLineStart(538)
+                .hasColumnStart(5)
+				.hasCategory("Logic error")
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [File contains tab characters (this is the first instance).](https://app.codacy.com/manual/uhafner/analysis-model/pullRequest?prid=4824671)